requirements.txt

```diff
@@ -1,17 +1,24 @@
+aiohttp==3.8.5
+aiosignal==1.3.1
 astroid==2.15.5
+async-timeout==4.0.2
+attrs==23.1.0
 cachetools==3.1.1
-certifi==2023.7.22
```

Comment:
Why did we regress here?  Google foolishness?

Reply:
probably something I did on my end

Reply:
fixed